bpf/utils.h

```diff
@@ -358,4 +358,14 @@ static inline void fill_in_others_protocol(flow_id *id, u8 protocol) {
     id->transport_protocol = protocol;
 }
 
+static inline bool is_transport_protocol(u8 protocol) {
+    switch (protocol) {
+    case IPPROTO_TCP:
+    case IPPROTO_UDP:
+    case IPPROTO_SCTP:
```

Comment:
Looking into the linux headers / `nf_conntrack_man_proto`, sounds like DCCP can also have port?

Reply:
but we don't support or parse that protocol in openshift

Reply:
yeah never mind .. no point doing it in xlat if we don't do it in the main path anyway